src/main/java/roboy/dialog/states/expoStates/PersonalInformationAskingState.java

```diff
@@ -0,0 +1,292 @@
+package roboy.dialog.states.expoStates;
```

Comment:
looks like copy-paste from src/main/java/roboy/dialog/states/**ordinaryStates**/PersonalInformationAskingState .java
please merge both states, we don't want to have duplicate code

Reply:
Not copypaste

Reply:
Open for further development. Should stay

Reply:
ok, they are indeed different. I got confused by the name.